modules/apps/dispatch/dispatch-test/src/testFunctional/tests/SwitchToTalendTabInJobScheduler.testcase

```diff
@@ -0,0 +1,49 @@
+@component-name = "portal-dispatch"
```

Comment:
How about name the testcase filename to TalendJobScheduler.testcase, and the name you are using right now I think is more suitable for a real case name :smile:

Reply:
agree